pkg/collector/wallet_balance_collector.go

```diff
@@ -58,12 +61,32 @@ func (wb WalletBalanceCollector) Collect(ch chan<- prometheus.Metric) {
 				walletBalance,
 				prometheus.GaugeValue,
 				balance,
-				[]string{account.Address, (*wb.RPCs)[account.ChainName].ChainID, account.Denom, status}...,
+				[]string{account.Address, (*wb.RPCs)[account.ChainName].ChainID, account.Denom, status, strings.Join(account.Tags, ",")}...,
```

Comment:
tags must defineable in the configs e.g.

```
accounts:
  - address: archway1avwvqzu9gv86g5fxx5p2xqe0w33wklt27jusdrhszwccnfnxx0rsmzz8nu
    chainName: archway
    denom: aarch
    tags: tag1,tag2
```

Reply:
my bad, it expect a list of strings

accounts:
  - address: archway1avwvqzu9gv86g5fxx5p2xqe0w33wklt27jusdrhszwccnfnxx0rsmzz8nu
    chainName: archway
    denom: aarch
    tags: ["tag1","tag2"]